runtime/fresh/routes/render.tsx

```diff
@@ -106,11 +113,13 @@ export const handler = async (
   try {
     page = await ctx.state.resolve(
       { ...section, ...props },
-      { resolveChain },
+      resolveChain ? { resolveChain } : undefined,
       original,
     );
   } catch (err) {
     if (err instanceof HttpError) {
+      throw new Error("I Don't know what im doing");
```

Comment:
this is desirable? what is the use case? what happened?

Reply:
done